src/components/Menu/Menu.module.css

```
@@ -21,4 +21,30 @@
     display: flex;
     align-items: center;
   }
+
+  ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px) !important;
+  }
+
+  ul[role*=menu] li ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px + var(--spacing-padding-md, 12) * 1px) !important;
+  }
+
+  ul[role*=menu] li ul[role*=menu] li ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px + var(--spacing-padding-md, 12) * 2px) !important;
+  }
+}
+
+.primary {
+  ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--shape-size-lg, 24) * 1px) !important;
+  }
+
+  ul[role*=menu] li ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--shape-size-lg, 24) * 1px + var(--spacing-padding-md, 12) * 1px) !important;
+  }
+
+  ul[role*=menu] li ul[role*=menu] li ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--shape-size-lg, 24) * 1px + var(--spacing-padding-md, 12) * 2px) !important;
+  }
```

Puoi controllare che con questa regola vada? 
Localmente io lo vedo come ci aspettiamo.

è da settare inline inlineIndent dentro il componente menu di antd

```suggestion
  ul[role*=menu] {
    padding-left: 12px !important;
  }

  div[role*=presentation] {
    padding-left: 0 !important;
  }
```

---

I have tried, but these changes do not give the same results for the following reasons:
- The elements do not have internal padding (icons are attached to the left border)
- The elements do not start from the beginning of the row but from where the external padding ends: in fact, the more nested the elements are the "shorter" they are
- The padding of categories is small compared to what we want

---

Solved in https://github.com/mia-platform/design-system/pull/355 ✅